src/TikzGraphs.jl

```diff
@@ -63,7 +63,9 @@ function plot{T<:AbstractString}(g::LightGraphs.SimpleGraph; layout::Layouts.Lay
     for e in LightGraphs.edges(g)
         a = src(e)
         b = dst(e)
-        print(o, "$a ->")
+        # print arrows only for directed graphs
```

Comment:
absolutely -  I don't have access to a proper computer right now but will recommit ASAP. Thanks!